### PR TITLE
chore: mirror player.src on the demo page demo page

### PR DESCRIPTION
**Pull request by brandonocasey**
## Description
Uses sourceset to mirror the source that was set via `player.src` in the ui so that it will be saved on page reload!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -88,6 +88,10 @@ <h3>Options</h3>
       <input id=override-native type="checkbox" checked>
       Override Native (reloads player)
     </label>
+    <label>
+      <input id=mirror-source type="checkbox" checked>
+      Mirror sources from player.src (reloads player)
```
Comment (gkatsev) [inline index.html:93]: Should we just call this `sourceset`? This the target of these pages are us, we can be as technical as we want.

**Review by gkatsev — APPROVED** [review]
Naming aside, LGTM

File: index.html (hunk 1 above)
Comment (brandonocasey) [reply index.html:93]: At first I wanted to, but that isn't really the feature that we care about here. We care about mirroring sources from `player.src`, `sourceset` just happens to make that very easy to do.

Comment (gkatsev) [reply index.html:93]: Maybe adding `uses sourceset` in the parens too.
My thought process here is that we always want mirroring but that it's possible that sourceset may cause issues so we'd want to disable it.

Comment (brandonocasey) [reply index.html:93]: I will do that and merge

**Review by gkatsev — APPROVED** [review]

**Pull request merged**